.env.example

```diff
@@ -0,0 +1,2 @@
+MONGODB_URI=#set your mongodb url here
```

Comment:
I guess you can remove this file, because its just an example :)

Reply:
I had a note that Thomas said it would be considered "best practice" to leave a file like this. On first sight, I´d had the same thought.